lsp-java.el

```diff
@@ -1122,7 +1126,7 @@ PROJECT-URI uri of the item."
                            ("registerOptions" (ht ("watchers"
                                                    (vector (ht ("globPattern" "**/*.java"))
                                                            (ht ("globPattern" "**/pom.xml"))
-                                                           (ht ("globPattern" "**/*.gradle"))
+                                                           (ht ("globPattern" "**/*.gradle{,.kts}"))
```

Comment:
It is more about the testing - you may keep this line(although I am not sure whether this pattern works) and then modify the kts file(e. g. add new dependency) and then:

1. Verify that lsp-java is sending update to the server
2. Verify that jdt-ls is updating the workspace(you will start to receive notifications).

If 2) is not happening we have to file jdtls bug.

Reply:
Nothing happens with the line as written.
Should we consider this a bug?

As separate lines, however, both directions work:
- `*lsp-log*` shows `workspace/didChangeWatchedFiles` notifications
- `~/.emacs.d/workspace/.metadata/.log` shows `!MESSAGE >> workspace/didChangeWatchedFiles`
  However, when I comment out a dependency, information about classes from it still show up, so I'm not sure of the effect.

Reply:
IMO it is a bug. Can you check whether the change is applied if you restart the jdt ls server? If it works, that means the problem is in jdt-ls which does not process the file notification.

Reply:
No change: commented out dependency still resolves after `lsp-restart-workspace`.
eclipse/eclipse.jdt.ls#449 seems to indicate Kotlin build file support is missing.